text/0000-add-author.md

```diff
@@ -0,0 +1,59 @@
+# Meta
+[meta]: #meta
+- Name: Add Author to RFC Metadata
+- Start Date: 2020-03-14
+- CNB Pull Request: https://github.com/buildpacks/rfcs/pull/64
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This is a proposal to add an `Author(s):` field to the `Meta` section of the RFC template.
+
+# Motivation
+[motivation]: #motivation
+
+When an RFC is merged, and then renamed, the commit history is lost. This makes it difficult to remember who wrote the RFC. Sometimes, an RFC is written by more than one person, but the linked PR will not always show this.
+
+# What it is
+[what-it-is]: #what-it-is
+
+A new `Author(s):` field in the Meta section of the RFC template.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+You're not going to believe this, but you add an `Author(s):` field to the Meta section of the RFC template.
+
+We will also rename the `CNB Pull Request` to `RFC Pull Request`.
```

Comment:
I added both the RFC Pull Request and the CNB Pull Request to this item: https://github.com/buildpacks/rfcs/blob/master/text/0005-contractual-build-plan.md

Should we drop one, keep it the same, or just make it more clear. I agree that having the discussion linked is important.

Reply:
I've changed this to leave the `CNB ...` fields alone. we can address those in another RFC.